src/components/Buttons/Button/Button.jsx

```diff
@@ -14,10 +22,12 @@ Button.propTypes = {
   title: PropTypes.node,
```

Comment:
I guess the title type is wrong )

Reply:
```suggestion
  title: PropTypes.string.isRequired,
```
I fixed everything. Thanks.